dali/test/python/test_fw_iterators.py

```diff
@@ -146,6 +149,103 @@ def test_mxnet_iterator_not_fill_last_batch_pad_last_batch():
     assert len(next_img_ids_list_set) == data_size
     assert len(set(next_mirrored_data)) == 1
 
+def test_gluon_iterator_last_batch_no_pad_last_batch():
```

Comment:
Can you add some test to check DENSE_TAG and SPARSE_TAG as well?

Reply:
I already did here https://github.com/NVIDIA/DALI/pull/1683/files#diff-50cb25d057baa7e4b3e08ec20cdd2d2cR226

Reply:
Missed that